Library/Homebrew/cli/named_args.rb

```diff
@@ -208,10 +208,22 @@ def to_paths(only: parent&.only_formula_or_cask, recurse_tap: false)
         end.uniq.freeze
       end
 
+      sig { returns(T::Array[Keg]) }
+      def to_keg
```

Comment:
Would it be wrong or more confusing to name it something like this? (default might not be the right word I didn't look at the logic of `resolve_keg` super closely)

```suggestion
      def to_default_kegs
```

The plural here and in the other `to_*` methods is because it can return multiple kegs if multiple named arguments are present. It essentially uses `map` to map arguments to whatever type we're looking for. So `to_keg` is a bit of a misnomer if it can return multiple items.

Reply:
I was thinking about changing the name to something like that. I definitely agree. Didn't think about changing the method names for `to_formulae_and_casks`, but I agree there as well.